src/bin/tikv-server.rs

```diff
@@ -62,6 +62,7 @@ use tikv::import::{ImportSSTService, SSTImporter};
 use tikv::pd::{PdClient, RpcClient};
 use tikv::raftstore::coprocessor::CoprocessorHost;
 use tikv::raftstore::store::{self, new_compaction_listener, Engines, SnapManagerBuilder};
+use tikv::server::http_server::HttpServer;
```

Comment:
prefer just calling it status server, since its configuration is already "status_addr".